assignments/arrays.js

```diff
@@ -75,22 +81,34 @@ console.log();
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for(let i = 0; i < inventory.length; i++) {
+  carModels.push(inventory[i].car_model);
+}
+carModels.sort();
+
+console.log(carModels);
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (var i = 0; i < inventory.length; i++) {
```

Comment:
- please make sure you're not using `var` anymore. Use `let` or `const`. Since we're changing `i` everytime (adding to it) we would use `let`.